src/Uno.Toolkit.UI/Controls/ZoomContentControl/ZoomContentControl.xaml

```diff
@@ -0,0 +1,85 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:ctl="using:Uno.Toolkit.UI">
```

Comment:
Since the component were moved to Toolkit.UI, would be nice to change this namespace code to utu:  no?
Just to keep the pattern.